### PR TITLE
re-add missing v1 filters.

### PR DESCRIPTION
**Pull request by jctanner**
#### What is this PR doing:
Somehow the final refactor got rid of some important filters.

#### Reviewers must know:

**PR Author & Reviewers**: Keep or remove backport labels per [Backporting Guidelines](https://github.com/ansible/galaxy_ng/wiki/Backporting-Guidelines)
**Reviewers**: Look for sound code, no [code smells](https://www.codegrip.tech/productivity/everything-you-need-to-know-about-code-smells/), docs & test coverage
**Merger**: When merging, include the Jira issue link in the squashed commit

### CODE REVIEW

File: galaxy_ng/tests/integration/api/test_community.py [hunk 1]
```diff
@@ -365,3 +365,47 @@ def test_v1_sync_with_user_and_limit(ansible_config):
...
+    assert isinstance(resp, dict)
+    assert resp.get('task') is not None
+    wait_for_v1_task(resp=resp, api_client=api_client)
+
+    # start the second sync
+    pargs = json.dumps({"github_user": github_user2, "limit": 10}).encode('utf-8')
+    resp = api_client('/api/v1/sync/', method='POST', args=pargs)
+    assert isinstance(resp, dict)
+    assert resp.get('task') is not None
+    wait_for_v1_task(resp=resp, api_client=api_client)
```
Comment (awcrosby) [inline galaxy_ng/tests/integration/api/test_community.py:398]: is the second sync used for anything in the test? I see only `github_user` is cleaned up at end.

File: galaxy_ng/tests/integration/api/test_community.py [hunk 2]
```diff
@@ -365,3 +365,47 @@ def test_v1_sync_with_user_and_limit(ansible_config):
...
+
+    # query by user
+    resp = api_client(f'/api/v1/roles/?owner__username={github_user}')
+    assert resp['count'] > 0
+    usernames = sorted(set([x['username'] for x in resp['results']]))
+    assert usernames == [github_user]
+
+    # validate autocomplete search only finds the relevant roles
+    sresp = api_client(f'/api/v1/roles/?autocomplete={github_user}')
+    assert sresp['count'] == resp['count']
```
Comment (awcrosby) [inline galaxy_ng/tests/integration/api/test_community.py:408]: `sresp` = second response?

**Review by awcrosby — COMMENTED** [review]
minor questions

File: galaxy_ng/tests/integration/api/test_community.py (hunk 2 above)
Comment (jctanner) [reply galaxy_ng/tests/integration/api/test_community.py:408]: yes.

File: galaxy_ng/tests/integration/api/test_community.py (hunk 1 above)
Comment (jctanner) [reply galaxy_ng/tests/integration/api/test_community.py:398]: It ensures that incorrect results don't come back from the search.

**Pull request merged**